Readme.md

```diff
@@ -101,6 +101,14 @@ The [Aragon OSx contracts](https://github.com/aragon/osx/tree/develop/packages/c
 
 The [contract-ethers](https://github.com/aragon/osx/tree/develop/packages/contracts-ethers) package is the NPM package that provides `ethers.js` wrappers to use the [Aragon OSx contracts](https://github.com/aragon/osx/tree/develop/packages/contracts).
 
+## Audit
```

Comment:
Do we want to add info about our other audit?

not priority and something we can do afterwards as well, but just checking.

Reply:
It's indirectly included as part of this one :+1: